ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -267,5 +268,21 @@ Item {
                 onCountChanged: currentIndex = 0
             }
         }
+
+        Connections {
+            enabled: suggestionsDialog.visible
+            target: root.rootStore.contactsStore.mainModuleInst
+            onResolvedENS: {
+                if (resolvedPubKey === "")
+                    return
+
+                const contactDetails = Utils.getContactDetailsAsJson(resolvedPubKey, false)
+
+                if (contactDetails.isBlocked || contactDetails.publicKey === root.rootStore.contactsStore.myPublicKey)
+                    return;
+
+                suggestionsDialog.visible = false
```

Comment:
It will break the binding, `suggestionsDialog` will no longer popup in the same instance of selector.

Reply:
that's ok. If you close "Send contact request" popup, it will close "Start chat" too

Reply:
If you have a test scenario, where it is not working, please, provide me the repro steps

Reply:
> that's ok. If you close "Send contact request" popup, it will close "Start chat" too

That's not ok. The component should **never** assume how it is used.

> If you have a test scenario, where it is not working, please, provide me the repro steps

Currently, there is no such scenario because _it just happens_ "Start chat" view is closed each time the dialog is closed. But it doesn't mean it won't change, and when it happens then we'll have a much more serious bug than "no results" displayed in the background.

Simple example of when it may get broken:

https://user-images.githubusercontent.com/33099791/205622437-1146a75f-81d0-4e8f-85e4-ab373fed170c.mp4

Reply:
refactored